src/routes/clinical-proxy.ts

```diff
@@ -0,0 +1,48 @@
+import urlJoin from 'url-join';
+import { CLINICAL_SERVICE_ROOT, SUBMISSION_TEMPLATE_PATH } from '../config';
+import logger from '../utils/logger';
+import express from 'express';
+import { Request, Response } from 'express';
+const router = express.Router();
+
+import { createProxyMiddleware } from 'http-proxy-middleware';
+
+// Our specification download service can't use GraphQL because GraphQL specification requires the content-type
+// that it returns be json, and we want to be able to return other content types, such as tab-separated-values,
+// so that the user is automatically prompted to save the file from their browser.
+
+// 'all' will retrieve the zip file with all templates
+// for specific templates 'templateName'.tsv or 'templateName' will get the tsv from clinical
+const templatePathRewrite = (pathName: string, req: Request) => {
+  const name = req.params.template.replace(/.tsv$/, '');
+  return urlJoin(SUBMISSION_TEMPLATE_PATH, name);
+};
```

Comment:
this and `programTsvExportPathRewrite` aren't really being reused anywhere, can just inline to avoid having to jump around

Reply:
yea, I guess they are small enough to inline eh. will update